playwright/runner.ts

```diff
@@ -7,6 +7,7 @@ import { enableLogs } from "../lib/config";
 import { AutoAction } from "../lib/types";
 
 const testRegion = (process.env.REGION || "NA").trim();
+test.describe.configure({ mode: 'parallel' });
```

Comment:
nice! How faster is it now and did you see any side-effects from it?

Reply:
This just allows parallelization within a single test file. This is important for this new spec because it's defining all the tests in a single file. For other tests, I guess the gain is primarily when running a single spec.